### PR TITLE
Feature/visualize schema

### PR DESCRIPTION
**Pull request by duncanjbrown**
## Context

I was trying to review #107 and I realised I had no idea what the schema looked like, so I generated a picture of it, and now I'm putting the picture in the README along with a way to update it

## Changes proposed in this pull request

Introduce a `generate_erd` management command, which depends on `django_extensions` and a couple of graphing libraries — those are added in `requirements-dev.txt` and only called on when `ENVIRONMENT=LOCAL`.

Add the image to `README.md`

Refactor a bunch of slightly gnarly checks for `ENVIRONMENT=LOCAL` to use a friendlier API: `HostingEnvironment.is_local()`

I didn't spend time making this generate automatically when running a migration locally (and I think that's a bit more work, as envs will differ in having ☝️ the necessary libraries installed, and I don't think there's a standard way to extend default management commands)... but I did add instructions for how to generate the diagram next to it in the README.

## Guidance to review

- try and run it locally — will require an install from `requirements-dev.txt`
- How's the `HostingEnvironment.is_local()` interface?
- is it displaying in the README?

## Link to JIRA ticket

N/A

## Things to check

- [X] I have added any new ENV vars in all deployed environments

### CODE REVIEW

**Comment by duncanjbrown** [comment]
I think [this failure](https://github.com/i-dot-ai/evaluation-registry/actions/runs/7299737258/job/19893125477?pr=110) is probably because `ENVIRONMENT=LOCAL` on CI?

**Comment by duncanjbrown** [comment]
> I think [this failure](https://github.com/i-dot-ai/evaluation-registry/actions/runs/7299737258/job/19893125477?pr=110) is probably because `ENVIRONMENT=LOCAL` on CI?

it was.

File: evaluation_registry/settings_base.py [hunk 1]
```diff
@@ -7,9 +7,11 @@
 import environ
 from django.conf import settings
 
+from .hosting_environment import HostingEnvironment
+
 env = environ.Env()
 
-if env.str("ENVIRONMENT", None) != "LOCAL":
+if HostingEnvironment.is_local():
```
Comment (gecBurton) [inline evaluation_registry/settings_base.py:14]: :)

**Review by gecBurton — APPROVED** [review]
LGTM, but i had to run `brew install gprof2dot` first before it would work, could we add this to the README.md?

**Comment by duncanjbrown** [comment]
Interesting — `gprof2dot` seems unrelated! This works on my machine without it, so I don't think we should add to the docs.  I wonder how that came to be warned for. It's a good point, tho — `graphviz` itself is an external dependency listed in the [PyDot docs](https://pypi.org/project/pydot/).

I'll add instructions to install `graphviz` via system package manager.

**Pull request merged**